lookout/style/format/descriptions.py

```diff
@@ -83,9 +83,8 @@ def describe_sample(feature: BagFeature, values: ndarray) -> str:
     :param values: The values to describe.
     :return: A string that describe the values of this feature.
     """
-    active = flatnonzero(values)[0]
-    counts = ("%s: %d" % (feature.names[i], count) for i, count in zip(active, values[active]))
-    return ", ".join(counts) if len(active) else "∅"
+    active = flatnonzero(values)
+    return "{%s}" % ", ".join(feature.names[index] for index in active) if len(active) else "∅"
```

Comment:
NOOP_REPR should only represent the NOOP label (and indeed uses `∅` atm). Here I really want to represent the empty set (not in the NOOP label context). If we want that to have a special variable holding the representation of the empty set it means we should also have one for non-empty (`{`and `}`) and many other symbols. It's probably more readable to keep it as is. WDYT @EgorBu?

Reply:
I was confused by the same `"∅"` for `NOOP` class and here.
What do you mean by empty set? Why do we need it here?

Reply:
I mean empty set because we display the roles features for example as a set of activated (sub) features: `left.2.roles = {FUNCTION, ADD}` for example. When the feature has no (sub)feature activated, then we still have to display something so the empty set is the most fitting: `left.2.roles = ∅`.